food.go

```diff
@@ -104,7 +104,11 @@ func (f *Food) Install() error {
 	unarchiveOrCopy(cachedFilePath, barrelDir, u.Path)
 
 	// This is just a safety check to make sure that there's nothing there when we link the package.
-	f.Unlink(pkg)
+	err = f.Unlink(pkg)
+	if err != nil {
+		return fmt.Errorf("an error occurred while unlinking '%v' try running 'gofish unlink %v': %v", f.Name, err, f.Name)
```

Comment:
In general, it's best to keep CLI command suggestions out of the SDK. Separation of concerns and all.

Can you please return a concrete error type here like `UnlinkError`, and in `cmd/gofish/install.go`, can you check for that error type?

For example:

```
if err := food.Install(); err != nil {
    if _, ok := err.(*UnlinkError); ok {
        return fmt.Errorf("%s. Try running 'gofish unlink %s'", err.Error(), food.Name)
    }
    return err
}
```

https://blog.golang.org/go1.13-errors

Reply:
Please also use `%s` for string values.

Reply:
was the error message good or do you want something different for the wording?

Reply:
the error message looks good.